roles/conserver/files/socat_retry.sh

```
@@ -2,6 +2,6 @@
 reconnTimeOut=5
 TERM=vt100
 while /bin/true
-    do /bin/socat "$@" 2>/dev/null
-    sleep "$reconnTimeOut"
+    do /bin/socat $* 2>/dev/null
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)